File: Java/1043.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double A, B, C;
        double perimetro, area;

        A = sc.nextDouble();
        B = sc.nextDouble();
        C = sc.nextDouble();

        perimetro = (A + B + C);

        if (A + B > C && B + C > A && C + A > B) {
            System.out.printf("Perimetro = %.1f\n", perimetro);
        } else {
            System.out.printf("Area = %.1f\n", area = ((A + B) * C) / 2);
        }

    }

}

File: Python/1043.py
A , B , C = list (map(float,input().split()))

perimetro = A + B + C
area = (((A + B)* C) / 2)

if A + B > C and B + C > A and C + A > B:
    print (f"Perimetro = {perimetro:.1f}")

else:
    print (f"Area = {area:.1f}")